packages/workbox-sw/index.mjs

```diff
@@ -39,6 +65,58 @@ class WorkboxSW {
   clientsClaim() {
     self.addEventListener('activate', () => self.clients.claim());
   }
+
+  /**
+   * Load a Workbox module by passing in the appropriate module name.
+   *
+   * @param {string} moduleName
+   */
+  loadModule(moduleName) {
+    if (this._options.disableModuleImports) {
+      // This can't be a WorkboxError as we can't rely on workbox-core being
+      // loaded.
+      throw new Error(
+        `Attempted to load '${moduleName}' while disableModuleImports is true.`
+      );
+    }
+
+    importScripts(this._getImportPath(moduleName));
```

Comment:
As discussed, this could be something to do wrap in a `try`/`catch` and give the user some additional context about, but we can wait until we play around with dynamic loading further and explore the different failure possibilities.